auth/api/iam/api.go

```diff
@@ -155,8 +165,8 @@ func (r Wrapper) Routes(router core.EchoRouter) {
 			// The following URLs require a user session:
 			paths := []string{
 				"/oauth2/:subjectID/user",
-				"/oauth2/:did/authorize",
-				"/oauth2/:did/callback",
+				"/oauth2/:subjectID/authorize",
```

Comment:
in the other URLs they're called `:subject`, might be good to keep them constant

Reply:
Will switch to `subjectID` since `subject` might be confusing for people not linking it to *DID subject*